backend/cmd/api/main.go

```
@@ -272,6 +273,13 @@ func main() {
 		return
 	}
 
+	rdb := redis.NewClient(&redis.Options{
+		Addr:     config.Redis.Address,
+		Password: config.Redis.Password,
+		Username: config.Redis.Username,
+		DB:       config.Redis.Database,
+	})
```

We should do a ping here and fail loudly if it doesn't work (config).